Client/Frontend/Browser/BrowserViewController/BrowserViewController+WKNavigationDelegate.swift

```diff
@@ -247,6 +247,12 @@ extension BrowserViewController: WKNavigationDelegate {
                 }
             }
         }
+        
+        // U2F code below
+        if let tab = tabManager[webView] {
+            // U2F should only be available in secure contexts
+            tab.userScriptManager?.isU2FEnabled = webView.hasOnlySecureContent
```

Comment:
FYI, not sure if its better here, but we have KVO setup here:

https://github.com/brave/brave-ios/blob/38554971754cace94437d7aa8e98defcd61653ce/Client/Frontend/Browser/BrowserViewController.swift#L952-L961

Which you could use to update the `userScriptManager` as the `hasOnlySecureContent` changes on the fly